gh_pages/index.md

```diff
@@ -19,7 +19,7 @@ This is accomplished by including a simple badge or URL in your project's README
 * <a name="unsupported"></a>__Unsupported__ – The project has reached a stable, usable state but the author(s) have ceased all work on it. A new maintainer may be desired.
 * <a name="moved"></a>__Moved__ - The project has been moved to a new location, and the version at that location should be considered authoritative. This status should be accompanied by a new URL.
 
-These status descriptions and the URLs to the corresponding icons are also available in a [JSON file](/badges/latest/badges.json).
+These status descriptions and the URLs to the corresponding icons are also available in a [JSON file](/badges/latest/badges.json) or in a more formal ontology using SKOS, defined in a [JSON-LD file](/badges/latest/ontology.jsonld).
```

Comment:
It might be useful to link "SKOS" to https://www.w3.org/TR/skos-reference/ to provide context.

Reply:
Agreed! Added now in 1a1f87d454aad2cc2f85956304e53cd6fba3155d